### ISSUE TITLE
IPython.nbformat.current deprecated in IPython 3

### ISSUE THREAD

**Issue by astrofrog** [paulgb/runipy#56]
I am currently getting the following deprecation warning with IPython 3:

```
/Volumes/Raptor/miniconda3/envs/production/lib/python3.4/site-packages/IPython/nbformat/current.py:19: UserWarning: IPython.nbformat.current is deprecated.

- use IPython.nbformat for read/write/validate public API
- use IPython.nbformat.vX directly to composing notebooks of a particular version
```

The changes are actually quite significant - the notebook object returned by ``IPython.nbformat.read`` now no longer has a worksheets attribute.

**Comment by gabraganca** [comment]
@astrofrog I made a PR dealing with it. 

And it seems that this functionality is built-in in ```nbconvert``` now.(see #54).

**Comment by jakirkham** [comment]
This should be fixed in @gabraganca, #53. Closing.... If it's not, feel free to give a shout.
